### PR TITLE
Fix unittests pointing to the external package "gopkg.in/ldap.v2"

### PR DESCRIPTION
**Pull request by christian-roggia**
Some of the *_test.go files are pointing to the external package "gopkg.in/ldap.v2". This is not the expected behavior of unittests and it is not consistent with other unittests inside the library.

### CODE REVIEW

File: example_test.go [hunk 1]
```diff
@@ -25,15 +23,15 @@ func ExampleConn_Bind() {
 
 // ExampleConn_Search demonstrates how to use the search interface
 func ExampleConn_Search() {
-	l, err := ldap.Dial("tcp", fmt.Sprintf("%s:%d", "ldap.example.com", 389))
```
Comment (liggitt) [inline example_test.go]: find/replace victim: `"ldap.example.com"`

**Comment by liggitt** [comment]
should probably remove the `- go get gopkg.in/ldap.v2` from travis.yml as well

File: example_test.go (hunk 1 above)
Comment (christian-roggia) [reply example_test.go]: You are completely right, I will fix this immediately.

**Comment by liggitt** [comment]
LGTM, thanks for the cleanup

**Pull request merged**